qemu/tests/qemu_guest_agent_update.py

```diff
@@ -0,0 +1,151 @@
+import logging
+import time
+import re
+
+from avocado.utils import process
+
+from virttest import env_process
+from virttest import error_context
+from virttest import utils_misc
+from virttest import data_dir
+
+from qemu.tests.qemu_guest_agent import QemuGuestAgentBasicCheckWin
+
+
+class QemuGuestAgentUpdateTest(QemuGuestAgentBasicCheckWin):
+
+    @error_context.context_aware
+    def gagent_check_pkg_update(self, test, params, env):
+
+        def copy_pkg(qemu_ga_pkg, dst_path):
+            """
+            copy qemu-ga to a specify dir and rename.
+            src_path,E:\guest-agent\qemu-ga-xx.msi
+            dst_path,C:\qemu-ga\qemu-ga-xx.msi
```

Comment:
Pls modify the description in fixed format.

Reply:
I think it's ok, as test, params, env are very clear, no need explain more.

Reply:
I mean like this, and the params are 'qemu_ga_pkg' and 'dst_path'.
            """
           change iso for virtio-win
           :param cdrom_virtio: iso file
           """

Reply:
modified,thanks